CMSgov/mint-app
# [EASI-3705] Move state agency help question

# EASI-3705

## Changes and Description

- Frontend
  - Moved `agencyOrStateHelp` from page one of `opsEvalAndLearning` to page 2 of `generalCharacteristics`
  - Moved `agencyOrStateHelp` likewise in readonly
  - Updated queries to new task list location
  - Updated unit tests
  - Updated cypress test
  - Updated translation location
- Backend
  - GQL
  - Store SQL
  - Migration
  - Go data model updates
  - Resolver updates



## How to test this change

- Build containers off main
  - `git checkout main && scripts/dev down && scripts/dev up && git checkout feature/EASI-3705_backend_move_state_agency_help_question`
- Insert test data
  - See below
- Migrate data
  - `scripts/dev db:migrate`
- Assert that data has migrated from plan ops eval and learning to plan general characteristics
  - The following columns should be moved
    - agency_or_state_help
    - agency_or_state_help_other
    - agency_or_state_note
- Assert that the front end operates as expected, with the agency help question moved from Ops Eval & Learning to General Characteristics

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
